shell/browser/ui/views/frameless_view.cc

```diff
@@ -5,6 +5,8 @@
 #include "shell/browser/ui/views/frameless_view.h"
 
 #include "shell/browser/native_window_views.h"
+#include "shell/browser/ui/views/inspectable_web_contents_view_views.h"
+#include "ui/aura/window.h"
```

Comment:
Are we sure we need these?  They were removed as not needed in https://github.com/electron/electron/commit/aa23198ad8ed023a09bf6162f0e268f4aa0aa524.

Reply:
ah you're right - removing!